storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc

```diff
@@ -164,10 +162,15 @@ include::modules/lvms-deleting-cloned-volumes-in-single-node-openshift.adoc[leve
 //Limitations for creating volume snapshots and volume clones
 include::modules/lvms-limitations-for-creating-snapshots-and-clones.adoc[leveloffset=+1]
 
+// Uninstalling LVM Storage
+
+include::modules/lvms-uninstalling-logical-volume-manager-operator-using-openshift-web-console.adoc[leveloffset=+1]
+include::modules/lvms-uninstalling-logical-volume-manager-operator-using-rhacm.adoc[leveloffset=+1]
+
 //Must-gather
 include::modules/lvms-download-log-files-and-diagnostics.adoc[leveloffset=+1]
 
 [role="_additional-resources"]
-.Additional resources
+.Additional resources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
```

Comment:
i *think* this may be an accidental copy/paste?

Reply:
Yes. It's a blunder :(
Updated it.